spec/rails_helper.rb

```diff
@@ -85,20 +120,31 @@
     raise StandardError.new "\"#{example.full_description}\" in #{example.location} timed out."
   end
 
-  config.around :each, :disable_bullet do |example|
-    Bullet.raise = false
-    example.run
-    Bullet.raise = true
-  end
-
-  def pre_transition_aged_youth_age
-    Date.current - CasaCase::TRANSITION_AGE.years
-  end
+  # NOTE: not applicable currently, left to show how to skip bullet errrors
+  # config.around :each, :disable_bullet do |example|
+  #   Bullet.raise = false
+  #   example.run
+  #   Bullet.raise = true
+  # end
 
   config.around do |example|
     Capybara.server_port = 7654 + ENV["TEST_ENV_NUMBER"].to_i
     example.run
   end
 
-  config.filter_run_excluding :ci_only unless ENV["GITHUB_ACTIONS"]
+  config.filter_run_excluding :ci_only unless ci_environment
 end
+
+RSpec::Matchers.define_negated_matcher :not_change, :change
+
+Shoulda::Matchers.configure do |shoulda_config|
+  shoulda_config.integrate do |with|
+    with.test_framework :rspec
+    with.library :rails
+  end
+end
+
+WebMock.disable_net_connect!(
+  allow_localhost: true,
+  allow: "selenium_chrome:4444"
+)
```

Comment:
This was in a spec/support/ file (only this -- deleted the file, see below), I think it makes more sense here.